server-spi/src/main/java/org/keycloak/storage/clientscope/ClientScopeLookupProvider.java

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
This should be in server-spi-private, since it is not expected to be supported anytime soon

Reply:
I'd like to keep this one in `server-spi` due to the fact that `ClientScopeProvider` (`server-spi`) inherits from this interface. wdyt?